DirectEmployers/MyJobs
# [PD-3428] create testing for main reducer

## What's this PR do?
PR creates the testing suite for the main reducer inside of the analytics application

## Screenshots (if appropriate)
N/A

## Where should the reviewer start?
Start inside of the analytics folder for testing the specs

## How should this be manually tested?
Pull down the repository. Inside of the myjobs folder, run dkgg watch to start test and make sure they are all passing. You can also view the source code

## Any background context you want to provide?
